packages/email-petition/src/email-petition.tsx

```diff
@@ -44,7 +43,7 @@ console.log("campaign: " + CAMPAIGN_NAME);
 
 const CAPTCHA_SITE_KEY = "6LdiglcpAAAAAM9XE_TNnAiZ22NR9nSRxHMOFn8E";
```

Comment:
do you think it's best to share one key for all sites that use the petition component? or think it might make sense to pass in the captcha site key as a prop so each site could have its own?

Reply:
oh i guess the key is specific to the secret we use server-side too. so if we wanted to separate it, we'd have to configure it there too. meh.